hyperlane/testing/src/mock_validator_set.rs

```diff
@@ -139,7 +139,7 @@ impl MockValidatorSet {
             .validators
             .iter()
             .map(|sk| {
-                let signature = eth_utils::sign_digest(multisig_hash.into_inner(), sk);
+                let signature = eth_utils::sign_digest(multisig_hash.into_inner(), sk).unwrap();
```

Comment:
This `.unwrap` used to be in the `eth_utils` library. Now I've removed that `.unwrap` in `eth_utils`, the unwrap is moved to here (which is ok because this is a test).